BlueDBOnDisk/src/test/java/org/bluedb/disk/BlueDbOnDiskTest.java

```diff
@@ -77,22 +77,31 @@ public void test_initializeCollection_existing_correct_type() throws Exception {
 		assertNotNull(db.collectionBuilder(getTimeCollectionName(), TimeKey.class, TestValue.class).build());  // make sure it works the second time as well
 	}
 
-	@SuppressWarnings("unchecked")
+	@SuppressWarnings({ "unchecked", "deprecation" })
+	@Test
+	public void test_initializeCollection_old() {
+		insertAtTime(10, new TestValue("Bob"));
+		try {
+			db.initializeCollection(getTimeCollectionName(), TimeKey.class, TestValue.class, TestValue2.class);
+		} catch(BlueDbException e) {
+			fail();
+		}
+	}
+
 	@Test
 	public void test_initializeCollection_invalid_type() {
 		insertAtTime(10, new TestValue("Bob"));
 		try {
-			db.initializeCollection(getTimeCollectionName(), TimeKey.class, TestValue2.class);
+			db.initializeCollection(getTimeCollectionName(), TimeKey.class, TestValue2.class, Arrays.asList());
 			fail();
 		} catch(BlueDbException e) {
 		}
 	}
 
-	@SuppressWarnings("unchecked")
 	@Test
 	public void test_initializeCollection_invalid_key_type() {
 		try {
-			db.initializeCollection(getTimeCollectionName(), HashGroupedKey.class, TestValue.class);
+			db.initializeCollection(getTimeCollectionName(), HashGroupedKey.class, TestValue.class, Arrays.asList());

```

Comment:
Do we have tests using the builder instead of the deprecated method? I'd rather update most stuff to use the builder at some point. Doesn't have to be now.

Reply:
I think everything uses the builder except one test that is only there to get coverage.